tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -985,8 +985,6 @@ impl Core {
             .stats
             .tuned_global_queue_interval(&worker.handle.shared.config);
 
-        debug_assert!(next > 1);
```

Comment:
It sounds like this was triggered by setting it to 1? In that case, let's add a regression test for that case.